opencog/persist/zmq/atomspace/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ ADD_CUSTOM_COMMAND(
 )
 
 ADD_LIBRARY (zmqatoms SHARED
-	ZMQMessages.pb
+	ZMQMessages.pb.cc ZMQMessages.pb.h
```

Comment:
This should not be required; cmake automatically (should be automatically) attaching the needed .cc to the end of the name string ... are you getting an error due to this?